### DISCUSSION TITLE
addax的插件能在datax-web中使用吗

### DISCUSSION THREAD

**Discussion by baobao2016** [wgzhao/Addax#839] [Ideas]
addax的插件能在datax-web中使用吗

**Comment by wgzhao** [comment]
可以使用的

**Reply by baobao2016** [reply]
比如我想安装datax3.0.1+datax-web3.0.1,因为datax缺少我想的插件，我可以把addax的插件上传到datax里使用吗，然后使用datax-web来配置job与参数。 相当把二者结合在一起使用。不知是否可行？

**Reply by wgzhao** [reply]
Addax 4.0 之后的版本已经不和 DataX 兼容了，所以你只能选择其中一个版本

**Reply by baobao2016** [reply]
感觉Addax 4.0与DataX兼容会市场更大。
